packages/cli/src/GenericHelpers.ts

```diff
@@ -31,3 +37,36 @@ export async function validateEntity(
 }
 
 export const DEFAULT_EXECUTIONS_GET_ALL_LIMIT = 20;
+
+class StringWithNoXss {
+	@NoXss()
+	value: string;
+
+	constructor(value: string) {
+		this.value = value;
+	}
+}
+
+export async function validateRecordNoXss(record: Record<string, string>) {
```

Comment:
Assuming good enough until we have more streamlined payload validation in place. Else let me know.

Reply:
Not a big fan of this approach, but you are right that we first need a standard mechanism for payload validation. Maybe we could add a comment that this is only a temporary solution